shared/constants/wallets.js

```diff
@@ -466,31 +456,34 @@ const getSelectedAccount = (state: TypedState) => state.wallets.selectedAccount
 
 const getDisplayCurrencies = (state: TypedState) => state.wallets.currencies
 
-const getDisplayCurrency = (state: TypedState, accountID?: Types.AccountID) =>
-  state.wallets.currencyMap.get(accountID || getSelectedAccount(state), makeCurrency())
+const getDisplayCurrency = (state: TypedState, accountID: Types.AccountID) =>
+  state.wallets.currencyMap.get(accountID, makeCurrency())
+
+const getPayments = (state: TypedState, accountID: Types.AccountID) =>
+  state.wallets.paymentsMap.get(accountID, null)
 
-const getPayments = (state: TypedState, accountID?: Types.AccountID) =>
```

Comment:
no need for defaults for these functions, we almost always already have the selected account ID